src/Symfony/Component/Security/Core/Tests/Authentication/Token/AbstractTokenTest.php

```diff
@@ -184,11 +185,9 @@ public function testSetUser($user)
 
     public function getUsers()
     {
-        $user = $this->getMockBuilder('Symfony\Component\Security\Core\User\UserInterface')->getMock();
-        $advancedUser = $this->getMockBuilder('Symfony\Component\Security\Core\User\AdvancedUserInterface')->getMock();
+        $user = $this->getMockBuilder(UserInterface::class)->getMock();
```

Comment:
all the non required "string to ::class" CS changes should be reverted to fit our policy and help reduce future merge conflicts as much as possible

Reply:
I've changed _all_ usages in my changes to use the strings (in upcoming push). Due to how the diffing works, some of them are not displaying it correctly, as methods have been added/removed. By using the string everywhere, it should be less confusing for git as well.